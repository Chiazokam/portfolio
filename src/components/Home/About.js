import React from 'react';
import Container from '@material-ui/core/Container';

const About = () => {
    return (
        <section className='about-section'>
            <Container fixed>
                <div className='section-title'>Who I am...</div>
                <div className='about-content'>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. Sed sagittis, sapien eget porta 
                    semper, sem sem imperdiet augue, eget elementum enim turpis a lorem. Praesent id nisi odio. Aliquam commodo 
                    nibh vitae quam pharetra ultricies. Praesent et vehicula ante. Nunc at erat libero. Nullam quis cursus nisl. Maecenas 
                    placerat auctor leo, vel maximus orci hendrerit id. Curabitur vitae vulputate odio. Nulla facilisi.</p>

                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. Sed sagittis, sapien eget porta semper, 
                    sem sem imperdiet augue, eget elementum enim turpis a lorem. Praesent id nisi odio. Aliquam commodo nibh vitae quam 
                    pharetra ultricies. Praesent et vehicula ante. Nunc at erat libero. Nullam quis cursus nisl. Maecenas placerat auctor leo, vel 
                    maximus orci hendrerit id. Curabitur vitae vulputate odio. Nulla facilisi.</p>
                </div>
            </Container>
        </section>
    )
}

export default About;
