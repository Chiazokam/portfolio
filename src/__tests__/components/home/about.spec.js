import React from 'react';
import { shallow } from 'enzyme';
import About from '../../../components/Home/About';

describe('About component', () => {
    it('renders successfully', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
    })
});
