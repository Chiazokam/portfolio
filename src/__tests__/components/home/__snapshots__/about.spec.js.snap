// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders successfully 1`] = `ShallowWrapper {}`;
